pyproject.toml

```diff
@@ -109,6 +109,9 @@ python_files = [
 testpaths = [
     "tests",
 ]
+markers = [
+    "disable_autouse"
+]
```

Comment:
I think most markers are in `pytest_configure` instead

Reply:
We could, but they're tests for the same hook. That's why I think putting them in the same class might make more sense. But move it to a separate class would work as well (or should we just move it outside as a standalone method)

Reply:
I moved it to `pytest_configure`. Can make it a separate class if needed

Reply:
There are already multiple instances where a class is tested in multiple test classes. The restriction does not really make sense.